app/assets/config/manifest.js

```diff
@@ -2,3 +2,4 @@
 //= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
+//= link_tree ../builds
```

Comment:
- [ ] When I try to run your project I get an error due to this. You need to either create the `app\assets\builds` folder (and create `.keep` file within it if you don't want git to ignore it) or remove this line. Besides this, you met other project requirements.

  ![image](https://github.com/Rifat8080/blog-app/assets/36542986/60f12435-9364-4073-91bb-6ee878fa3b0f)